src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -202,6 +224,16 @@ public SFArrowResultSet(
     }
   }
 
+  public SFArrowResultSet(
```

Comment:
should we depracate the old constructor?

Reply:
This is the same constructor

Reply:
Fixed